src/compiler/types.ts

```diff
@@ -7365,6 +7365,7 @@ export const enum ScriptKind {
 }
 
 export const enum ScriptTarget {
+    /** @deprecated */

```

Comment:
Should this also be marked internal ?

Reply:
I said this on https://github.com/microsoft/TypeScript/pull/57472#discussion_r1503210241, but I don't think we should remove them entirely, at least not until 6.0.